syntax/crisp_lexer.mll

```diff
@@ -19,49 +53,38 @@ Stack.push Crisp_syntax.min_indentation scope_stack;;
 let test_indentation indentation follow_on_tokens lexbuf =
```

Comment:
While the machinery is still fresh in your mind, it's useful to describe the role and intended behaviour of this updated function given its importance.